File: OnlineMetronomeREST/Models/Piece.cs
using System;
namespace OnlineMetronomeREST.Models
{
    public class Piece
    {

        public string PieceID { get; set; }
        public string PieceName { get; set; }
        public string PieceComposer { get; set; }

        public string PieceOwner {get; set;}

        public Piece(){}
    }
}

File: OnlineMetronomeREST/src/app/models/piece.ts
import { environment } from '../../environments/environment';
import { globalVars } from '../globalVars';
import { Measure } from '../models/measure';

export class Piece {
    pieceComposer:string;
    pieceName:string;
    userName:string;
    pieceMeasures: Array<Measure>

    constructor(composer:string, name:string, measures: Array<Measure>, userName: string) {
        this.pieceComposer = composer;
        this.pieceName = name;
        this.pieceMeasures = measures;
        this.userName = userName;
    }
}